upgrade-instructions.md

```diff
@@ -14,6 +14,41 @@ Note: this section is in progress. It will be updated during all the development
 
 Changes to apply between 3.3.x and 3.4.x will be reported here.
 
+Change list:
+
+ - [Upgrade to ElasticSearch 6.7](#upgrade-to-elasticsearch-6.7)
+
+#### Upgrade to ElasticSearch 6.7
```

Comment:
is there a way to inform that it concerns only the (experimental) james distributed product?